site/content/docs/5.2/migration.md

```diff
@@ -75,6 +75,8 @@ If you need more details about the changes, please refer to the [v5.2.3 release]
 
 - <span class="badge bg-success">New</span> Form helpers icon buttons will help you to provide extra help information with the `.form-helper` utility class.
 
+- <span class="badge bg-success">New</span> Small size form controls are now available. Small inputs are defined by their `.form-control-sm`, small selects by their `.form-select-sm`. `.input-group-sm` can be used depending on the context while they can also be associated with `.col-form-label-sm` to correctly follow their size if needed.
```

Comment:
@hannahiss: I've added more info in this sentence to explain all new classes that have been added in this PR. Please check this out and suggest improvement if needed :)

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1230/commits/6d1f964d6f72971432a10c22dbc17e77a17cb131